src/animation/AnimationClip.js

```diff
@@ -326,7 +326,7 @@ Object.assign( THREE.AnimationClip, {
 			} else {
 				// ...assume skeletal animation
 
-				var boneName = '.bones[' + bones[ h ].name + ']';
+				var boneName = '.bones[' + bones[ hierarchyTracks[h].parent ].name + ']';
```

Comment:
Is this change correct?
I think this change means `boneName` should be parent bone's name instead of bone's name.
It's very far from "AnimationClip uses id instead index".